### PR TITLE
Align assembly with eclipse CHE latest version

### PR DESCRIPTION
**Pull request by garagatyi**
@vparfonov @ashumilova Please review

### CODE REVIEW

File: assembly/assembly-main/src/assembly/webapps/ROOT/META-INF/context.xml [hunk 1]
```diff
@@ -13,4 +13,4 @@
 -->
```
Comment (vparfonov) [inline assembly/assembly-main/src/assembly/webapps/ROOT/META-INF/context.xml]: we can remove it

File: assembly/assembly-machine-server/src/assembly/webapps/ROOT/WEB-INF/rewrite.config [hunk 2]
```diff
@@ -0,0 +1 @@
+RewriteRule ^/api/ext/(.*)$     /ide/ext/$1    [L]
```
Comment (vparfonov) [inline assembly/assembly-machine-server/src/assembly/webapps/ROOT/WEB-INF/rewrite.config]: don't need any more

Comment (garagatyi) [inline assembly/assembly-machine-server/src/assembly/webapps/ROOT/WEB-INF/rewrite.config]: Is it not needed only for artik or for che too?

File: assembly/assembly-main/src/assembly/webapps/ROOT/META-INF/context.xml (hunk 1 above)
Comment (garagatyi) [inline assembly/assembly-main/src/assembly/webapps/ROOT/META-INF/context.xml]: Is it not needed only for artik or for che too?

Comment (vparfonov) [inline assembly/assembly-main/src/assembly/webapps/ROOT/META-INF/context.xml]: yep, i will do it in che

**Comment by vparfonov** [comment]
ok

**Pull request merged**